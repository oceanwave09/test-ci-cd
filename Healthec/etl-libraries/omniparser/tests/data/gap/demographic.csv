~ROW1~,~Domain~,~Patient First Name~,~Patient Last Name~,~Patient Date of Birth~,~External Patient ID~,~External Practice ID~,~First Name~,~Last Name~,~Date of Birth~,~Middle Name~,~Suffix~,~Gender~,~Race Code~,~Race Name~,~Race Dictionary~,~Ethnicity Code~,~Ethnicity Name~,~Ethnicity Dictionary~,~Primary Language Code~,~Primary Language Name~,~Primary Language Dictionary~,~Marital Status Code~,~Martial Status Name~,~Marital Status Dictionary~,~SSN~,~Active~,~Primary Physician First Name~,~Primary Physician Last Name~,~Primary Physician NPI~,~Override PCP~,~Other Physician First Name~,~Other Physician Last Name~,~Other Physician NPI~,~Remove Spec Other Physician~,~Home Phone~,~Other Phone~,~Email~,~Home Address Line1~,~Home Address Line2~,~City~,~State~,~Zip~,~Patient/Provider Partnership~,~Patient/Prov Partnershp Date~,~Advance Directives~,~Advance Directives Date~,~Inactive Reason Code~,~Inactive Reason Name~,~Inactive Reason Dictionary~,~Health Literacy~,~Care Manager's First Name~,~Care Manager's Last Name~,~Care Manager's NPI~,~Remove Specified Care Manager~,~Date of Death~,~Patient Reminder Preference~,~Patient Last Reminder Date~,~MIHIN Common Key~
~1~,~Demographics~,~First7660601~,~Last7660601~,~19600101~,~7660601~,~Epic~,~First7660601~,~Last7660601~,~19600101~,~M~,~~,~F~,~4~,~White or Caucasian~,~~,~8~,~Non Hispanic/Latino~,~~,~91~,~English~,~~,~80~,~Widowed~,~~,~365383111~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~3018984501~,~~,~NoReply@HealthEC.COM~,~343 Thornall Street~,~~,~Edison~,~NJ~,~08837~,~10~,~20230105~,~0~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU~
~2~,~Demographics~,~First7660602~,~Last7660602~,~19580101~,~7660602~,~Epic~,~First7660602~,~Last7660602~,~19580101~,~M~,~~,~F~,~4~,~White or Caucasian~,~~,~8~,~Non Hispanic/Latino~,~~,~91~,~English~,~~,~30~,~Married~,~~,~362487638~,~~,~~,~~,~207R00000X~,~~,~~,~~,~~,~~,~3018984502~,~~,~NoReply@HealthEC.COM~,~343 Thornall Street~,~~,~Edison~,~NJ~,~08837~,~10~,~20211220~,~0~,~20150728~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV~
~3~,~Demographics~,~First7660603~,~Last7660603~,~19920101~,~7660603~,~Epic~,~First7660603~,~Last7660603~,~19920101~,~M~,~~,~F~,~4~,~White or Caucasian~,~~,~8~,~Non Hispanic/Latino~,~~,~91~,~English~,~~,~10~,~Divorced~,~~,~306306346~,~~,~~,~~,~207Q00000X~,~~,~~,~~,~~,~~,~3018984503~,~~,~~,~343 Thornall Street~,~~,~Edison~,~NJ~,~08837~,~10~,~20220411~,~0~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF~
~4~,~Demographics~,~First7660604~,~Last7660604~,~19890101~,~7660604~,~Epic~,~First7660604~,~Last7660604~,~19890101~,~~,~~,~M~,~4~,~White or Caucasian~,~~,~8~,~Non Hispanic/Latino~,~~,~91~,~English~,~~,~60~,~Single~,~~,~382044384~,~~,~~,~~,~207R00000X~,~~,~~,~~,~~,~~,~3018984504~,~~,~~,~343 Thornall Street~,~~,~Edison~,~NJ~,~08837~,~10~,~20221004~,~0~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R~
~5~,~Demographics~,~First7660605~,~Last7660605~,~19690101~,~7660605~,~Epic~,~First7660605~,~Last7660605~,~19690101~,~~,~~,~M~,~4~,~White or Caucasian~,~~,~8~,~Non Hispanic/Latino~,~~,~91~,~English~,~~,~30~,~Married~,~~,~382466529~,~~,~~,~~,~207R00000X~,~~,~~,~~,~~,~~,~3018984505~,~~,~NoReply@HealthEC.COM~,~343 Thornall Street~,~~,~Edison~,~NJ~,~08837~,~10~,~20210913~,~0~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~~,~RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2~
