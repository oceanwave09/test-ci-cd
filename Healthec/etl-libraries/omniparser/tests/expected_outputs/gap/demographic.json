[
	{
		"advance_directives": "0",
		"caremanager_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"city": "Edison",
		"dob": "b699c2b9c5b08522dc986a2def4aca6e",
		"domain": "Demographics",
		"email": "491960e89aae91c76c9cc97b168b400eed8e767455442a3b2e778b2f81e40784",
		"ethnicity_code": "8",
		"ethnicity_name": "Non Hispanic/Latino",
		"external_patient_id": "7660601",
		"external_practice_id": "Epic",
		"first_name": "920b1bfa8aeb6237faf53f11643c8ecf",
		"gender": "F",
		"home_address_line1": "c440a0835375182d48fba64a5feff05c45bcf2885247b6eb17c886d34511408e",
		"home_address_line2": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"home_phone": "b31742a1c5636a664da696f97b46a5b9",
		"last_name": "41c337237eef08cdc7d9f625231d112a",
		"marital_status_code": "80",
		"martial_status_name": "Widowed",
		"middle_name": "4a1447984f29706cf4d20d9c588c8f0a",
		"other_phone": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"patient_dob": "b699c2b9c5b08522dc986a2def4aca6e",
		"patient_first_name": "920b1bfa8aeb6237faf53f11643c8ecf",
		"patient_id": "NCKMK1Y42Y1W2TZIURJ1QI1MG4OY4EN5FA6QMRJU",
		"patient_last_name": "41c337237eef08cdc7d9f625231d112a",
		"patient_provider_partnership": "10",
		"patient_provider_partnership_date": "20230105",
		"primary_language_code": "91",
		"primary_language_name": "English",
		"primary_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"race_code": "4",
		"race_name": "White or Caucasian",
		"row": "1",
		"row_id": "62be6bcc-3cbe-43c6-9200-eeffee8d5746",
		"ssn": "024b27270a66cca5da13cf5bbe2567be",
		"state": "NJ",
		"zip": "08837"
	},
	{
		"advance_directives": "0",
		"advance_directives_date": "20150728",
		"caremanager_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"city": "Edison",
		"dob": "56a973e9f56eb206d960f2974ef9804b",
		"domain": "Demographics",
		"email": "491960e89aae91c76c9cc97b168b400eed8e767455442a3b2e778b2f81e40784",
		"ethnicity_code": "8",
		"ethnicity_name": "Non Hispanic/Latino",
		"external_patient_id": "7660602",
		"external_practice_id": "Epic",
		"first_name": "4369a95df3b5152bdd7c64a72efd1aa6",
		"gender": "F",
		"home_address_line1": "c440a0835375182d48fba64a5feff05c45bcf2885247b6eb17c886d34511408e",
		"home_address_line2": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"home_phone": "0d3111ab4bc29921db046883ab5dd319",
		"last_name": "fc65ea507d571ca1dde7d800f2f95573",
		"marital_status_code": "30",
		"martial_status_name": "Married",
		"middle_name": "4a1447984f29706cf4d20d9c588c8f0a",
		"other_phone": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"patient_dob": "56a973e9f56eb206d960f2974ef9804b",
		"patient_first_name": "4369a95df3b5152bdd7c64a72efd1aa6",
		"patient_id": "S176Y2EGTDK6TACILF4P1TG72BCMXQERYWSNG6KV",
		"patient_last_name": "fc65ea507d571ca1dde7d800f2f95573",
		"patient_provider_partnership": "10",
		"patient_provider_partnership_date": "20211220",
		"primary_language_code": "91",
		"primary_language_name": "English",
		"primary_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_npi": "08fc2ff053d2df3f09a9dbff07cb1925",
		"race_code": "4",
		"race_name": "White or Caucasian",
		"row": "2",
		"row_id": "81fea49b-cc0a-418d-ad93-025f004ce381",
		"ssn": "d79be78309031b32be499bf5ec8f1b66",
		"state": "NJ",
		"zip": "08837"
	},
	{
		"advance_directives": "0",
		"caremanager_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"city": "Edison",
		"dob": "65d8491e60430a0736b2abfd79856d4d",
		"domain": "Demographics",
		"email": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"ethnicity_code": "8",
		"ethnicity_name": "Non Hispanic/Latino",
		"external_patient_id": "7660603",
		"external_practice_id": "Epic",
		"first_name": "0430717827ce4c57e07c1e445bc70477",
		"gender": "F",
		"home_address_line1": "c440a0835375182d48fba64a5feff05c45bcf2885247b6eb17c886d34511408e",
		"home_address_line2": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"home_phone": "fac9b5e0648bac81ea41880503dc12d7",
		"last_name": "bd39484c0baafaf573a91d7eeb470750",
		"marital_status_code": "10",
		"martial_status_name": "Divorced",
		"middle_name": "4a1447984f29706cf4d20d9c588c8f0a",
		"other_phone": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"patient_dob": "65d8491e60430a0736b2abfd79856d4d",
		"patient_first_name": "0430717827ce4c57e07c1e445bc70477",
		"patient_id": "DN7M0T0SA7U6WWYWSO5VYE7Q2ZFBT5O4JN534OIF",
		"patient_last_name": "bd39484c0baafaf573a91d7eeb470750",
		"patient_provider_partnership": "10",
		"patient_provider_partnership_date": "20220411",
		"primary_language_code": "91",
		"primary_language_name": "English",
		"primary_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_npi": "ccccf3189645fdd9634c6948c6c71043",
		"race_code": "4",
		"race_name": "White or Caucasian",
		"row": "3",
		"row_id": "ab21e013-9215-429b-a732-2b5ae9ccc159",
		"ssn": "073e3286c8ff018de91c393e67a83a39",
		"state": "NJ",
		"zip": "08837"
	},
	{
		"advance_directives": "0",
		"caremanager_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"city": "Edison",
		"dob": "9fde4afa3f81b3e57465ff6668b68b41",
		"domain": "Demographics",
		"email": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"ethnicity_code": "8",
		"ethnicity_name": "Non Hispanic/Latino",
		"external_patient_id": "7660604",
		"external_practice_id": "Epic",
		"first_name": "4c2170080510d285b48e195f9c690c5a",
		"gender": "M",
		"home_address_line1": "c440a0835375182d48fba64a5feff05c45bcf2885247b6eb17c886d34511408e",
		"home_address_line2": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"home_phone": "03c0522a5e4213d320fa5b486e40b3a4",
		"last_name": "29244f7cbc96db65581dba3839e472fe",
		"marital_status_code": "60",
		"martial_status_name": "Single",
		"middle_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_phone": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"patient_dob": "9fde4afa3f81b3e57465ff6668b68b41",
		"patient_first_name": "4c2170080510d285b48e195f9c690c5a",
		"patient_id": "IBXEP0B1Y6MBH02112RG567IJZQPF7QEES7KKY5R",
		"patient_last_name": "29244f7cbc96db65581dba3839e472fe",
		"patient_provider_partnership": "10",
		"patient_provider_partnership_date": "20221004",
		"primary_language_code": "91",
		"primary_language_name": "English",
		"primary_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_npi": "08fc2ff053d2df3f09a9dbff07cb1925",
		"race_code": "4",
		"race_name": "White or Caucasian",
		"row": "4",
		"row_id": "54167320-8bc2-44fe-af51-2dfef0c528fa",
		"ssn": "ceb6646cf2ed0f3174acf2a6473b2919",
		"state": "NJ",
		"zip": "08837"
	},
	{
		"advance_directives": "0",
		"caremanager_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"caremanager_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"city": "Edison",
		"dob": "6d65a93dd15e1ce35deca05d2f90426c",
		"domain": "Demographics",
		"email": "491960e89aae91c76c9cc97b168b400eed8e767455442a3b2e778b2f81e40784",
		"ethnicity_code": "8",
		"ethnicity_name": "Non Hispanic/Latino",
		"external_patient_id": "7660605",
		"external_practice_id": "Epic",
		"first_name": "fe7030703fd7abee786ea532bf04cd7a",
		"gender": "M",
		"home_address_line1": "c440a0835375182d48fba64a5feff05c45bcf2885247b6eb17c886d34511408e",
		"home_address_line2": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"home_phone": "6e80428e7e9e6dc6b7a0e558a9e249e6",
		"last_name": "4b4e3c02a281e755267bd96cc57327b0",
		"marital_status_code": "30",
		"martial_status_name": "Married",
		"middle_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_phone": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"other_physician_npi": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"patient_dob": "6d65a93dd15e1ce35deca05d2f90426c",
		"patient_first_name": "fe7030703fd7abee786ea532bf04cd7a",
		"patient_id": "RN7EVM4S7Z7499FJUZLM7RRCM2L2JT8CDGWOQZB2",
		"patient_last_name": "4b4e3c02a281e755267bd96cc57327b0",
		"patient_provider_partnership": "10",
		"patient_provider_partnership_date": "20210913",
		"primary_language_code": "91",
		"primary_language_name": "English",
		"primary_physician_first_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_last_name": "20a5c6f47975465f0f3a05f8cbbf2ce8",
		"primary_physician_npi": "08fc2ff053d2df3f09a9dbff07cb1925",
		"race_code": "4",
		"race_name": "White or Caucasian",
		"row": "5",
		"row_id": "351d0d92-a410-464d-b5e8-eab95eb4dbfc",
		"ssn": "b2d1ffc42bd12d6c0a21ab4a85ba2b4d",
		"state": "NJ",
		"zip": "08837"
	}
]
